sources/sdk/k8s-operator/src/operator.js

```diff
@@ -20,13 +23,23 @@ const defaultHttpApiFactory = () =>
     response.end('default backend')
   }
 
-const getAuthBearer = header => {
-  if (!(header && header.startsWith('Bearer '))) {
-    throw new OperatorError('Invalid token')
+const makeAuthToken = name => ({
+  signedCookie(req) {
+    return req.signedCookies[name]
+  },
+  authHeader(req) {
+    const authorization = req.get('authorization')
+    return authorization.substring(7, authorization.length)
```

Comment:
What if `authorization` is undefined ?

What if the header does not start with `"Bearer "` ?